workflow/scripts/oncoprint.py

```
@@ -162,11 +163,32 @@ def sort_oncoprint_labels(data):
             # target vector: label values, converted into factors
             target_vector = labels_df.loc[label]
             # ignore any NA in the target vector and correspondingly remove the rows in the feature matrix
-            not_na_target_vector = target_vector[~pd.isna(target_vector)].astype("category")
+            not_na_target_vector = target_vector[~pd.isna(target_vector)]
+            
+            target_is_numeric = pd.api.types.is_numeric_dtype(not_na_target_vector)
+            breakpoint()
```

Ruff (0.15.15)

[error] Trace found: `breakpoint` used

(T100)